Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

```diff
@@ -67,107 +64,139 @@ import androidx.compose.ui.unit.Dp
+                tabs.forEach { section ->
+                    val selected = section == currentSection
+                    val tint by animateColorAsState(
+                        if (selected) {
+                            JetsnackTheme.colors.iconInteractive
+                        } else {
+                            JetsnackTheme.colors.iconInteractiveInactive
+                        }
+                    )
 
-                JetsnackBottomNavigationItem(
-                    icon = {
-                        Icon(
-                            imageVector = section.icon,
-                            tint = tint,
-                            contentDescription = null
-                        )
-                    },
-                    text = {
-                        Text(
-                            text = stringResource(section.title).toUpperCase(
-                                ConfigurationCompat.getLocales(
-                                    LocalConfiguration.current
-                                ).get(0)
-                            ),
-                            color = tint,
-                            style = MaterialTheme.typography.button,
-                            maxLines = 1
-                        )
-                    },
-                    selected = selected,
-                    onSelected = { onSectionSelected(section) },
-                    animSpec = springSpec,
-                    modifier = BottomNavigationItemPadding
-                        .clip(BottomNavIndicatorShape)
-                )
+                    JetsnackBottomNavigationItem(
+                        icon = {
+                            Icon(
+                                imageVector = section.icon,
+                                tint = tint,
+                                contentDescription = null
+                            )
+                        },
+                        text = {
+                            Text(
+                                text = stringResource(section.title).toUpperCase(
+                                    ConfigurationCompat.getLocales(
+                                        LocalConfiguration.current
+                                    ).get(0)
+                                ),
+                                color = tint,
+                                style = MaterialTheme.typography.button,
+                                maxLines = 1
+                            )
+                        },
+                        selected = selected,
+                        onSelected = {
+                            if (section.route != currentRoute) {
+                                navController.navigate(section.route) {
+                                    // Keep the FEED section on the back stack, so the user
+                                    // navigates back to it first before quitting the app.
+                                    popUpTo = navController
```

Comment:
This can be replaced with `navController.graph.startDestination` I think

Reply:
Nope. Because we're using nested navigation, the `startDestination` will actually be defined by the outer graph, instead of the inner (nested) one.

Reply:
In that case you want this: https://github.com/chrisbanes/tivi/blob/main/app/src/main/java/app/tivi/home/Home.kt#L78

Reply:
Fixed!